app/config.go

```diff
@@ -46,11 +46,20 @@ func (a *App) EnvironmentConfig() map[string]interface{} {
 	return a.Srv().EnvironmentConfig()
 }
 
+func (s *Server) UpdateConfigWithoutDesanitize(f func(*model.Config)) {
```

Comment:
This is the new way to update things without desanitization (only used by my code in the `app/server.go` file). Here is the only place we set to `true` the `Set` parameter in the config store.